app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+    before_action :authenticate_user!,  except: [:new, :create]
```

Comment:
Why you exclude those two operation from authentication?

Reply:
> Why you exclude those two operation from authentication?

In the time of creating a user we can't apply authentication